### ISSUE TITLE
Vulkan port by ANdriusRome/Megai2

### ISSUE THREAD

**Issue by dalerank** [memononen/nanovg#576]
src here https://github.com/dalerank/nanogui/tree/master/src/vulkan

**Comment by porky11** [comment]
But that's not a NanoVG port. It's not even written in C. How can this help here?

**Comment by dalerank** [comment]
yep, sorry, https://github.com/dalerank/nanogui/blob/master/include/nanogui/nanovg_vk.h
